<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>textAnimatedMainWindow</class>
 <widget class="QMainWindow" name="textAnimatedMainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1025</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TextAnimated</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>depositphotos_54186841-stock-illustration-beautifully-decorated-english-alphabets-letter.jpg</normaloff>depositphotos_54186841-stock-illustration-beautifully-decorated-english-alphabets-letter.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="animatedButton">
    <property name="geometry">
     <rect>
      <x>212</x>
      <y>10</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Анимировать</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>671</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Наберите текст, который нужно анимировать...</string>
    </property>
   </widget>
   <widget class="QLabel" name="figureAnimatedLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Фигуры анимации</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>191</width>
      <height>691</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="figures_widget_hidden">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>189</width>
       <height>689</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>760</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Сохранить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="skrinButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>760</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Скрин</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cleanButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>760</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exampleButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>760</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Пример</string>
    </property>
   </widget>
   <widget class="QPushButton" name="historyButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>760</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>История</string>
    </property>
   </widget>
   <widget class="QPushButton" name="spravkaButton">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>760</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Справка</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="squareFigureCheckBox">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>90</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Прямоугольники</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="circleFigureCheckBox">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>120</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Круги</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="triangleFigureCheckBox">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>150</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Треугольники</string>
    </property>
   </widget>
   <widget class="QLabel" name="figureLabel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>60</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Фигуры</string>
    </property>
   </widget>
   <widget class="QLabel" name="sizeLabel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>180</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Размер фигур</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="sizeBigRadioButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>200</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Большой</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">sizeButtonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="sizeBetweenRadioButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>230</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Средний</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">sizeButtonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="sizeSmallRadioButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>260</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Маленький</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">sizeButtonGroup</string>
    </attribute>
   </widget>
   <widget class="QLabel" name="strLabel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>290</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Двустрочный</string>
    </property>
   </widget>
   <widget class="QLabel" name="animationLabel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>340</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Анимация</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="increaseAnimatedCheckBox">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>360</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Увеличение</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="rotationAnimatedCheckBox">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>390</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Вращение</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="flyAnimatedCheckBox">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>420</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Полёт</string>
    </property>
   </widget>
   <widget class="QLabel" name="palitraLabel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>480</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Палитра</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="palitraMonoRadioButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>510</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Однотонная</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">countColorsButtonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="palitraPoliDuoRadioButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>540</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Двуцветная</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">countColorsButtonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="palitraPoliTrioRadioButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>570</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Трехцветная</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">countColorsButtonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="palitraPoliRadioButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>600</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Многоцветная</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">countColorsButtonGroup</string>
    </attribute>
   </widget>
   <widget class="QCheckBox" name="nonAnimatedCheckBox">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>450</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Без анимации</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="saveAsButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>760</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Сохранить как...</string>
    </property>
   </widget>
   <widget class="QComboBox" name="strComboBox">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>310</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Нет</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Да</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="openButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>760</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Открыть</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea_3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>60</y>
      <width>671</width>
      <height>621</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="widget_hidden">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>669</width>
       <height>619</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="prevStepButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>690</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/back.png</normaloff>icons/back.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>690</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/play-button.png</normaloff>icons/play-button.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pauseButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>690</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/pause.png</normaloff>icons/pause.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="nextStepButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>690</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/next.png</normaloff>icons/next.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="upSpeedButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>730</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>x2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="downSpeedButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>730</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>÷2</string>
    </property>
   </widget>
   <widget class="QSlider" name="slider">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>690</y>
      <width>511</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="tickPosition">
     <enum>QSlider::NoTicks</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>730</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="countStepsLabel">
    <property name="geometry">
     <rect>
      <x>504</x>
      <y>730</y>
      <width>511</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="currentStepLabel">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>730</y>
      <width>511</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Текущий кадр:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="countColorsButtonGroup"/>
  <buttongroup name="sizeButtonGroup"/>
 </buttongroups>
</ui>
